### PR TITLE
legal: developer's certificate of origin 1.1

### PR DESCRIPTION
**Pull request by polymo1**
### Changes

- Added Developer's Certificate of Origin 1.1

This certificate affirms the right of contributors, whilst giving SideStore the right to use contributions for use regarding the SideStore project.

### CODE REVIEW

**Review by SternXD — APPROVED** [review]

**Review by JoeMatt — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 Thank you for your interest in contributing to SideStore! SideStore is a community driven project, and it's made possible by people like you.
 
+By contributing to this Project (SideStore), you agree to the Developer's Certificate of Origin found in [AGREEMENT.md](./AGREEMEND.md). Any contributions to this project after the addition of the Developer's Certificate of Origin are subject to its policy.
```
Comment (JJTech0130) [inline CONTRIBUTING.md:5]: Typo (links to nonexistent AGREEMEND.md)

Comment (polymo1) [reply CONTRIBUTING.md:5]: Cant currently fix it, you're welcome to (if able). It's on a branch not a fork.

**Review by stossy11 — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 Thank you for your interest in contributing to SideStore! SideStore is a community driven project, and it's made possible by people like you.
 
+By contributing to this Project (SideStore), you agree to the Developer's Certificate of Origin found in [AGREEMENT.md](AGREEMENT.md). Any contributions to this project after the addition of the Developer's Certificate of Origin are subject to its policy.
```
Comment (junepark678) [inline CONTRIBUTING.md:5]: You could've made the file name more descriptive. I don't know how other projects do this but I'd rather the file name to be descriptive

File: AGREEMENT.md
Comment (junepark678) [inline AGREEMENT.md:1]: Again, this file

File: CONTRIBUTING.md (hunk 2 above)
Comment (junepark678) [reply CONTRIBUTING.md:5]: Something more along the lines of  `CERTIFICATE-OF-ORIGIN.md`

**Review by junepark678 — APPROVED** [review]

**Comment by polymo1** [comment]
@Spidy123222 @JJTech0130 @nythepegasus @lonkelle I would appreciate it if you all could put in your two cents. I would like to merge this soon :)

**Pull request merged**